server/src/main/java/org/elasticsearch/plugins/PluginsLoader.java

```diff
@@ -389,7 +406,7 @@ static String toModuleName(String name) {
         return result;
     }
 
-    static final String toPackageName(String className) {
+    static String toPackageName(String className) {
```

Comment:
Automatic cleanup (IJ complaining)

Reply:
I'm surprised that wasn't caught before.